bundles/net.tourbook/src/net/tourbook/data/TourSegment.java

```diff
@@ -22,9 +22,11 @@
    public int     serieIndex_Start;
    public int     serieIndex_End;
 
-   public int     time_Recording;
+   public int     time_Elapsed;
 
-   public int     time_Driving;
+   public int     time_Recorded;
+   public int     time_Paused;
+   public int     time_Moving;
    public int     time_Break;
    public int     time_Total;
```

Comment:
Add `_Device` or `_Computed` postfix

Reply:
Do you prefer : 
`timeDevice_Recorded`
or
`time_Recorded_Device`
?

Reply:
I would rename them similar like the fields in `TourData`:

````
int deviceTime_Elapsed;
int deviceTime_Recorded;
int deviceTime_Paused;

int computedTime_Moving;
int computedTime_Break;
````

Reply:
Fixed